app/layout.tsx

```diff
@@ -28,7 +29,7 @@ export default function RootLayout({
       <body
         className={`${geistSans.variable} ${geistMono.variable} antialiased`}
       >
-        {children}
+        <ApolloProvider>{children}</ApolloProvider>
```

Comment:
1 question , we use apolloProvider to pass client to all other pages , so as you have not passed any prop so is it required ? or it will work same without ApolloProvider as the backend is also integrated in the same project ?